### PR TITLE
Module 0, Taks 1. Very first commit

### PR DESCRIPTION
**Pull request by paveldpua**

### CODE REVIEW

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
+In this pull request I've added ...
+
+<!-- Please mention all relevant module/task numbers. -->
+
+Taks 1
+virtualenv
```
Comment (Pacu2) [inline .github/PULL_REQUEST_TEMPLATE.md]: This template provides data to prefill your Pull Request's request. It should be kept as generic as possible. Please use the template as provided. 

However, it's a good thing to keep your pull requests informative, the information you've entered here could be added manually while opening a pull request from the next task onwards.

You can read more about PR templates here https://blog.axosoft.com/enhancing-pull-request-descriptions-templates/

**Review by Pacu2 — CHANGES_REQUESTED** [review]
Please use pull request template as provided

File: .github/PULL_REQUEST_TEMPLATE.md (hunk 1 above)
Comment (paveldpua) [reply .github/PULL_REQUEST_TEMPLATE.md]: Thank you Filip.
I have reverted my changes to PR template

**Review by Pacu2 — APPROVED** [review]

File: .github/PULL_REQUEST_TEMPLATE.md (hunk 1 above)
Comment (Pacu2) [reply .github/PULL_REQUEST_TEMPLATE.md]: Approved, looks good!

**Pull request merged**